x-pack/plugins/ml/public/util/chart_utils.js

```diff
@@ -228,6 +228,17 @@ export function getTickValues(startTimeMs, tickInterval, earliest, latest) {
   return tickValues;
 }
 
+export function getXTransform(t) {
+  const regexResult = /translate\(\s*([^\s,)]+)([ ,]([^\s,)]+))?\)/.exec(t);
+  if (Array.isArray(regexResult) && regexResult.length >= 2) {
+    // return as number
+    return +regexResult[1];
+  }
```

Comment:
Curious how we feel as a team about using `Number(regexResult[1])` here instead of `+` operator for converting strings to numbers? They do the same thing but I think using `Number` is more explicit and readable. It is a bit slower than using `+` but this type of thing isn't really going to be a bottleneck for performance.